node/node_test.go

```diff
@@ -146,3 +146,122 @@ func TestService_Delete(t *testing.T) {
 		})
 	}
 }
+
+func TestService_DeleteAll(t *testing.T) {
+	t.Parallel()
+	tests := []struct {
+		name                    string
+		preparePodServiceMockFn func(m *mock_node.MockPodService)
+		prepareFakeClientSetFn  func() *fake.Clientset
+		wantErr                 bool
+	}{
+		{
+			name: "delete all nodes and pods on nodes",
```

Comment:
Ah, Do you want to delete only Pods on node? (= do you want not to delete unscheduled pods?)
Yes, I think it is better than deleting all nodes and all pods.

Currently, `DeleteAll` deletes all nodes and all pods (include unscheduled pods.), so, should be fixed.

Reply:
I'll fixed it.